### ISSUE TITLE
D_real

### ISSUE THREAD

**Issue by mytrjp** [wiseodd/generative-models#60]
Hi wiseodd,
I am a little confused . Why you are returning D_prob and hence D_real although you are not using ? I know logits is enough to calculate the loss but why an extra discriminator output you want.

Thanks
